frontend/.env

```diff
@@ -5,3 +5,5 @@ REACT_APP_BACKEND_URL=https://simple-report-qa-api.app.cloud.gov/graphql
 REACT_APP_OKTA_URL=https://simplereport.gov/
 REACT_APP_APPINSIGHTS_KEY=ec65665e-358e-4f07-9292-3d80f23bcc3b
 REACT_APP_PATIENT_EXPERIENCE_ENABLED=false
+REACT_APP_V1_ACCESS_CONTROL_ENABLED=false
+REACT_APP_V2_ACCESS_CONTROL_ENABLED=false
```

Comment:
v1 - changing user roles. This should be true once the backend is wired in (which is dependent on it being completed first)
v2 - changing facilities. This is a WIP